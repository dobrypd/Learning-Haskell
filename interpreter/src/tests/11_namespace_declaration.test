namespace ns1 {
    int fun1() {
        return 10;
    }
}

int main() {
    return;
}
