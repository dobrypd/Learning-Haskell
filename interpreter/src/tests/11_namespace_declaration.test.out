Welcome to Kappa interpreter
Undefined
[(fromList [],fromList [(Ident "main",([],[SStm (JumpS SjumpFour)]))],fromList [(Ident "ns1",fromList [(Ident "fun1",([],[SStm (JumpS (SjumpFive (Econst (Eint 10))))]))])])]
